core/services/flux_monitor.go

```diff
@@ -27,6 +28,9 @@ const defaultHTTPTimeout = 5 * time.Second
 // Monitor supports.
 const MinimumPollingInterval = models.Duration(defaultHTTPTimeout)
 
+// MinimumIdleThreshold is the minimum allowed value for the IdleThreshold attribute.
+const MinimumIdleThreshold = models.Duration(defaultHTTPTimeout)
```

Comment:
Any reason for this choice of default? Seems arbitrary.

Reply:
True, I've updated it to be greater than or equal to the pollingInterval